test/server/rules.bzl

```diff
@@ -14,9 +14,10 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
+load("@graknlabs_dependencies//builder/java:rules.bzl", "host_compatible_java_test")
 
 def grakn_java_test(name, native_grakn_artifact, deps = [], classpath_resources = [], data = [], **kwargs):
-    native.java_test(
+    host_compatible_java_test(
```

Comment:
Oh? Does this work?

What are the implications of this change from `java_test` to `host_compatible_java_test`?

Reply:
Now that I think about it it's wrong to use `host_compatible_java_test` here. Therefore I've reverted it back to `native.java_test`.

The reason why I think `native.java_test` is more correct is because in practice, `grakn_java_test`  it expects the caller to already select the right artifact wit the use of `native_grakn_artifact`.